src/util/expr.cpp

```diff
@@ -81,13 +103,21 @@ void exprt::copy_to_operands(
   op.push_back(e3);
 }
 
+/// Create a \ref typecast_exprt to the given type.
+/// \param _type: cast destination type
+/// \deprecated use constructors instead
```

Comment:
:bulb: There is a corresponding comment in expr.h that can be deleted.

Reply:
deleted